esp/services/ws_workunits/ws_workunitsHelpers.cpp

```diff
@@ -4233,8 +4233,13 @@ void CWsWuFileHelper::readWULogToFiles(IConstWorkUnit *cwu, WsWuInfo &winfo, con
     ILogAccessFilter *logFetchFilter = getJobIDLogAccessFilter(wuid);
     zapLogFilterOptions.logFilter.logFetchOptions.setFilter(logFetchFilter);
 
-    VStringBuffer componentLog("%s%c%s-log.%s", path, PATHSEPCHAR, wuid, logfileextension.str());
-    readWULogToFile(componentLog, winfo, zapLogFilterOptions);
+    if (zapLogFilterOptions.includeRelatedLogs)
+    {
+        VStringBuffer componentLog("%s%c%s-log.%s", path, PATHSEPCHAR, wuid, logfileextension.str());
+        readWULogToFile(componentLog, winfo, zapLogFilterOptions);
+    }
+    if (!zapLogFilterOptions.includePerComponentLogs)
+        return;
```

Comment:
not sure I understand the logic here..
The includeRelatedLogs block (which is on by default).. gathers everything that matches wuid ?

includePerComponentlogs (default off), walks through <Process> and individually queries for a log per pod, correct?

Why would be want both on?


In bare-metal we have 2 options:
1) Gather all logs except thorworkers logs (default)
2) Gather all related logs and thorworker logs (if IncludeSlaveLogs is checked, off by default).

There are many thor workers, so collecting them all is a lot by default, and we shouldn't be doing that.
Most other logs are probably fine to though.

I think we (the UI) should collect with a select list of available components (derivable from the workunit metadata), where thor workers is not checked by default.

That may be too big a change right now, so I'm okay with something simple for now.